filters/diag/diag.go

```diff
@@ -146,6 +167,59 @@ func (r *random) Request(ctx filters.FilterContext) {
 
 func (r *random) Response(ctx filters.FilterContext) {}
 
+func (r *repeat) Name() string { return RepeatName }
+
+func (r *repeat) CreateFilter(args []interface{}) (filters.Filter, error) {
+	if len(args) != 2 {
+		return nil, filters.ErrInvalidFilterParameters
+	}
+
+	text, ok := args[0].(string)
+	if !ok || text == "" {
+		return nil, filters.ErrInvalidFilterParameters
+	}
+
+	var len int64
+	switch v := args[1].(type) {
+	case float64:
+		len = int64(v)
+	case int:
+		len = int64(v)
+	case int64:
+		len = v
+	default:
+		return nil, filters.ErrInvalidFilterParameters
+	}
+	if len < 0 {
+		return nil, filters.ErrInvalidFilterParameters
+	}
+
+	return &repeat{[]byte(text), len}, nil
+}
+
+func (r *repeat) Request(ctx filters.FilterContext) {
+	ctx.Serve(&http.Response{
+		StatusCode: http.StatusOK,
+		Header:     http.Header{"Content-Length": []string{strconv.FormatInt(r.len, 10)}},
```

Comment:
I have added `Content-Length` which eliminates `Transfer-Encoding: chunked` and improves transfer speed significantly for large response:
```
$ bin/skipper -inline-routes='* -> repeatContent("x", 10000000000) -> <shunt>' -access-log-disabled
```
without content length:
```
$ curl localhost:9090 -o /dev/null
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 9536M    0 9536M    0     0   729M      0 --:--:--  0:00:13 --:--:--  724M
```
with content length:
```
$ curl localhost:9090 -o /dev/null
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 9536M  100 9536M    0     0  1945M      0  0:00:04  0:00:04 --:--:-- 1964M
```